Gamma_Analysis.py

```diff
@@ -0,0 +1,339 @@
+"""
+This will perform a gamma analysis
+It will collect two spectra:
+M = Measurement Spectrum
+B = Background Spectrum
+"""
+from __future__ import print_function
+from SpectrumFileBase import SpectrumFileBase
+import SPEFile
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import os
+# exec('Isotope_identification.py')
+# Use bottom line for Python 3 only. Use top comment for Python 2.7 only.
+exec(open('Isotope_identification.py').read())
+
+
+EFFICIENCY_CAL_COEFFS = [5.1164, 161.65, 3952.3, 30908]
+
+
+def absolute_efficiency(energy, coeffs=EFFICIENCY_CAL_COEFFS):
+    """
+    Returns absolute efficiencies for a given set of energies, based on a
+    provided efficiency calibration.
+    """
+    efficiency = []
+    for i in range(len(energy)):
+        efficiency.append(math.exp(coeffs[3] *
+                          (math.log(energy[i])/energy[i])**3 -
+                          coeffs[2]*(math.log(energy[i])/energy[i])**2 +
+                          coeffs[1]*(math.log(energy[i])/energy[i]) -
+                          coeffs[0]))
+    return efficiency
+
+
+def emission_rate(net_area, efficiency, livetime):
+    """
+    this function returns the emission rate of gammas per second
+    alongside its uncertainty.
+    """
+    emission_rate = [net_area[0]/(efficiency*livetime),
+                     net_area[1]/(efficiency*livetime)]
+    return emission_rate
+
+
+def Isotope_Activity(Isotope, emission_rates, emission_uncertainty):
```

Comment:
You have an input argument called `Isotope` but through your `exec` call at the top of the file, you also have defined a class called `Isotope`. This is confusing because if you look at the variable in this function, you might think it refers to the `Isotope` class, and instead you have to look carefully at where else it is defined. If you change the `exec` call to `import ... as ii` then this is no longer an issue because the class is `ii.Isotope`.